fuckingsite_c3/src/App.js

```diff
@@ -121,14 +111,7 @@ class App extends Component {
         <h1> My todo-list </h1>
         <div className ="block_3">
           <TaskCreator onKeyUpHandler={this.createTask.bind((this))} />
-            {this.state.tasks.length}
-          
-            <ul className = "task--list">
-              {this.state.tasks.map((task,i) => {
-                  return <Task onClickHandler={this.removeTask.bind(this, i)} taskName={task.taskName} key = {"i"} />
-                  })
-              }
-            </ul>
+          <TaskList removeTask={(id) => this.removeTask(id)} tasks={this.state.tasks}/>
```

Comment:
the arrow function `{(id) => this.removeTask(id)}` removes the need to use bind and clearly states the data to be passed through